### ISSUE TITLE
Failed to get device IDError code: 7

### ISSUE THREAD

**Issue by amiltonrsilva** [zlgopen/awtk#568]
Hi. 

   How to fix that error code 7.

![Screenshot from 2021-06-27 10-19-16](https://user-images.githubusercontent.com/17910780/123545976-2fc8f300-d731-11eb-979a-1b6b1451dcd8.png)

Ubuntu 16.04.7 LTS
Release:	16.04

Thank you in advance.

**Comment by soyzhc** [comment]
Are you using VMware virtual machine？
AWTK Designer Ubuntu only support the IDE hard disk type，you can change the vm setting

**Comment by amiltonrsilva** [comment]
No. It's not in virtual machine, It is installed directly in a hard drive partition. 
This error occurs just after the application ask for authentication.

![image](https://user-images.githubusercontent.com/17910780/123581067-8249f400-d7b1-11eb-9c33-d7f10eb8816c.png)

**Comment by xuchaoze** [comment]
Can you try this command in Terminal, and tell me the result?
lsblk --nodeps -no serial

**Comment by amiltonrsilva** [comment]
lsblk --nodeps -no serial


191925446914















WD-WX41A59KEHTV

**Comment by soyzhc** [comment]
It may be a bug reading hard disk ID, please wait for the next upgrade version.
Please pay attention to awtk.zlg.cn
